api/src/main/scala/designated/director/repositories/DraftRepository.scala

```diff
@@ -16,5 +19,5 @@ case class DraftRepository(conn: Connection) extends Neo4jRepository[Draft](conn
   }
 
   val key: Draft => String = d => d.name.replaceAll(" ", "")
-  val insert: Draft => String = d =>  s"""{id:"${d.id}", name:"${d.name}", numOfRounds:${d.numOfRounds}, numOfTeams:${d.numOfTeams}}"""
+  val insert: Draft => String = d => d.toJson.compactPrint
```

Comment:
maybe standardize on one or the other.
```suggestion
  val insert: Draft => String = _.toJson.compactPrint
```